app/services/archive/frozen_cohort_profile.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Archive
+  class FrozenCohortProfile < UnvalidatedProfile
```

Comment:
I think the inheritance on these archivers is a bit misleading as the `FrozenCohortProfile/User` archiver _will_ archive validated profiles/users, unless I've missed something?

Reply:
I've pulled some common bits out into a module